lib/discourse_redis.rb

```diff
@@ -76,10 +76,11 @@ def mget(*args)
     DiscourseRedis.ignore_readonly { @redis.mget(*args) }
   end
 
-  def del(k)
+  def del(*keys)
     DiscourseRedis.ignore_readonly do
-      k = "#{namespace}:#{k}"  if @namespace
-      @redis.del k
+      keys = keys.flatten(1)
```

Comment:
is the flatten needed?

Reply:
It's needed if someone passes an array of keys like this `Discourse.redis.del(array_of_keys)` which the redis gem allows (see [here](https://github.com/redis/redis-rb/blob/21ec1dec064f43def1f47983431861e6491b770e/lib/redis/commands/keys.rb#L188-L193)). I think our API should match that of the redis gem.

Reply:
I see, needed for namespacing case, not needed for non namespacing case, but that is a rare call anyway.